docs/security/authorization/kibana-privileges.asciidoc

```diff
@@ -2,14 +2,31 @@
 [[kibana-privileges]]
 === Kibana privileges
 
-This section lists the Kibana privileges that you can assign to a role.
+Privileges have levels that you can use to manage which features users can access. Roles have privileges to determine whether users have write or read access.
+
+
+==== Privilege levels
+
+You can assign privileges the following levels:
+
+[horizontal]
+`base`::
+Grants access to all available features in Kibana (Discover, Visualize, Dashboard, and so on).
+
+`feature`::
+Grants access to a specific feature.
+
+For more information, consult the <<role-management-api-put, Role Management API>>
```

Comment:
@legrego Does the link to the Role Management API apply to both the privilege levels and role privileges?  If so, let's move this line as a separate paragraph under the intro statement.

Otherwise, the doc looks good.

Reply:
Yep, that link applies to both. I'll go ahead and move this to a separate paragraph. Thanks!